### DISCUSSION TITLE
Installation page - bloated

### DISCUSSION THREAD

**Discussion by TadasV1** [nuxt/nuxt#18566] [General]
Problem:
as an existing user, who starts new Nuxt3 project (coming from v2) AND:
1. does NOT need to Play
2. does NOT use Visual Studio
3. use ONLY npm

I find a bunch of irrelevant and distracting information.
Feels like Nuxt3 is different from what I need.

Had to go to npmjs to find install command. So in my case, the most of page is useless to me.

Also:
**Prerequisities** _are not really_ prerequisities, since prereq is a MUST and _implies that I can NOT start project without these_.
1. I dont NEED Visual Studio
2. I dont NEED Volar Extension

Maybe better:

**Explore with Sandboxes** (seperate page for sandoboxes) //for new users or existing and want to preview
**Install** //for different package managers
**Install for Visual Code Users** //most popular ide1, ide2 and so on
**Start New Project** 
**Start Development Server**
**Next Steps**

I believe it is _critical_ to avoid frustration and confusion on one of the first pages for the upcoming users.

**Comment by dsvgl** [comment]
From my perspective (experienced v2 user), the Getting Stared section is pretty neat (not too short and not too long). And it lists all the needed steps and commands to get started quickly. Also good for new users IMO. Not sure what you are missing. Could you elaborate?
It's always room for improvement, sure. And you are right about VS Code: it would be nice to also have instructions for other IDEs.

**Comment by TadasV1** [comment]
it just feels all is prepared for visual studio users. like i must transition or something. very unwelcoming.

some little explanation what is npm run dev **-- -o** (bold part)

a simple structure:
_Explore with Sandboxes //SHOULD BE @INTRODUCTION_

**Install** //for different package managers
**Install for Visual Code Users** //most popular ide1, ide2 and so on
**Start New Project**
**Start Development Server**
**Next Steps**

P.S. I cant figure what PLAY ONLINE has to do with INSTALLATION:/

**Reply by dsvgl** [reply]
Since this is Open Source and free, why not proposing your desired structure within a PR?
As I said, I'm with you about the IDE part. There could be a small comment about the `-- -o`part which is actually a good candidate for a small and simple PR.

> P.S. I cant figure what PLAY ONLINE has to do with INSTALLATION:/

C'mon
